src/components/tutorial-meta/index.tsx

```diff
@@ -80,7 +81,15 @@ export default function TutorialMeta({
 					to bookmark tutorials.
 				</Text>
 			) : null}
-			{variant ? <VariantList variant={variant} /> : null}
+			{variant ? (
+				<>
+					<VariantList variant={variant} className={s.variantList} />
+					<MobileVariantDropdownDisclosure
+						variant={variant}
+						className={s.mobileVariantDropdownDisclosure}
+					/>
+				</>
+			) : null}
```

Comment:
💭 Should the `VariantList` itself manage its mobile / desktop variations?  I like how they are split in two as they are wholly different presentationally, but perhaps the consumer (`tutorial-meta`) should not be responsible for this concern.

Reply:
I think I would do it this way myself:

- VariantList (thing exposed to consumers), and renders both, with the CSS logic here
  - DesktopVariantList
  - MobileVariantList
  
Not sure if any state management comes into play, but state management would also be managed at the `VariantList` level in my mind so when switching between desktop/mobile your selections are remembered.